omod/src/main/webapp/resources/htmlforms/RegimenChange.xml

```diff
@@ -95,11 +101,14 @@
                             <label><b>Current Regimen</b></label>
                         </td>
                         <td>
-                            <lookup complexExpression="#if ($fn.latestObs(90315,$encounterDate)) #set ($currentRegimen=$fn.latestObs(90315,$encounterDate)) #else #set ($currentRegimen=$fn.latestObs(99064,$encounterDate)) #end"/>
+                            <lookup complexExpression="#if ($fn.latestObs(90315,$previousRegimenDate)) #set ($currentRegimen=$fn.latestObs(90315,$previousRegimenDate)) #else #set ($currentRegimen=$fn.latestObs(99064,$previousRegimenDate)) #end"/>
```

Comment:
<img width="997" alt="Screenshot 2021-11-24 at 09 35 26" src="https://user-images.githubusercontent.com/3336745/143187203-477f87c9-11e0-41fc-94cb-08fd17c321e3.png">
<img width="1694" alt="Screenshot 2021-11-24 at 09 35 45" src="https://user-images.githubusercontent.com/3336745/143187212-1e0ccd3e-33f2-48bb-b5f9-154584e2d6fb.png">
@ssmusoke when you changed this see what happens to the current regimen placeholder

Reply:
@slubwama I do not understand this comment - what changed?

Reply:
@ssmusoke there are two images one showing a transfer regimen. And showing the regimen form that failed to render the Current Regimen.

Reply:
<img width="929" alt="Screenshot 2021-11-24 at 10 42 18" src="https://user-images.githubusercontent.com/3336745/143195696-cc249b7b-e88c-4dbd-afc3-c68e1db22149.png">
This is the one that renders when I un-deploy your changes.

Reply:
@slubwama Look at the dates, I had to change the implementation due to the fact that the regimen ended the previous date. So when you add the second one plus edit you would not get an accurate regimen 

In this case there is no transfer in regimen so you can change to it - so that you get a regimen line

Reply:
@slubwama I have worked on this issue please review again - Thanks